pytermgui/context_managers.py

```diff
@@ -8,26 +8,15 @@
 
 from __future__ import annotations
 
-from os import name
 from contextlib import contextmanager
-from typing import Callable, Generator, Any, Union, List
-
-from .ansi_interface import (
-    is_interactive,
-    save_cursor,
-    restore_cursor,
-    print_to,
-    show_cursor,
-    hide_cursor,
-    set_echo,
-    unset_echo,
-    set_alt_buffer,
-    unset_alt_buffer,
-    cursor_up,
-    report_mouse,
-    translate_mouse,
-    MouseEvent,
-)
+from os import name
+from typing import Any, Callable, Generator, List, Union
+
+from .ansi_interface import (MouseEvent, cursor_up, hide_cursor,
+                             is_interactive, print_to, report_mouse,
+                             restore_cursor, save_cursor, set_alt_buffer,
+                             set_echo, show_cursor, translate_mouse,
+                             unset_alt_buffer, unset_echo)
```

Comment:
This is the type of formatting change I don't quite understand. This is a file you didn't actually touch, and for me Black changes this into a much more readable format.